services/QuillLMS/spec/controllers/subscriptions_controller_spec.rb

```diff
@@ -101,6 +101,34 @@
     end
   end
 
+  context "with school admin" do
+    let!(:school1) { create(:school) }
+    let!(:school2) { create(:school) }
+    let!(:schools_admins1) { create(:schools_admins, school: school1) }
+    let!(:schools_admins2) { create(:schools_admins, user: schools_admins1.user, school: school2) }
+    let!(:subscription1) { create(:subscription, account_type: Subscription::SCHOOL_PAID ) }
+    let!(:school_subscription1) { create(:school_subscription, subscription: subscription1, school: school1) }
+    let!(:subscription2) { create(:subscription, account_type: Subscription::SCHOOL_PAID, expiration: '2020-01-1'.to_date ) }
+    let!(:school_subscription2) { create(:school_subscription, subscription: subscription2, school: school2) }
+
+    before { allow(controller).to receive(:current_user) { schools_admins1.user } }
+
+    describe '#school_admin_subscriptions' do
+      it 'should set the instance variables' do
+        get :school_admin_subscriptions, params: { :format => 'json' }
+        expect(JSON.parse(response.body)['user_associated_school_id']).to eq schools_admins1.user.school
```

Comment:
After confirming spec failure locally, I noticed that `schools_admins1.user.school` is nil.  Is this expected?